packages/ai/tests/ai/common/test_config.py

```
@@ -0,0 +1,117 @@
+
+
+# Ensure src is in path
+src_path = Path(__file__).parent.parent.parent.parent / 'src'
+if str(src_path) not in sys.path:
+    sys.path.insert(0, str(src_path))
+
+
+class MockIJson(dict):
+    """Mock IJson class that supports isinstance checks and dict methods."""
+
+    pass
```

Ruff (0.15.7)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)